tests/CMakeLists.txt

```diff
@@ -4,12 +4,14 @@ FetchContent_Declare(
     GIT_REPOSITORY https://github.com/catchorg/Catch2.git
     GIT_TAG        v2.13.0)
 FetchContent_MakeAvailable(Catch2)
+FetchContent_MakeAvailable(tree-sitter)
 
 add_executable(MiniLua-tests
     main.cpp
     unit_tests.cpp
     integration_tests.cpp
     tree-sitter.cpp)
+target_include_directories(MiniLua-tests PRIVATE ${tree-sitter_SOURCE_DIR}/lib/include)
```

Comment:
For a temporary fix this is fine but using Targets would be better.

I'm not sure why the `target_link_libraries(... INTERFACE Tree-Sitter)` does not work.

Reply:
I'm not sure what changes you propose here… the explicit path is bad but I could not find a workaround; it is strange though, as it works for the library without the explicit include path.